aleph_message/tests/test_models.py

```diff
@@ -237,3 +237,16 @@ def test_messages_from_disk():
                 console.print(message_dict)
                 console.print_json(e.json())
                 raise
+
+
+def test_invalid_messages():
+    """Invalid messages should not crash the entire generation of the MessagesResponse object."""
+    path = Path(
+        os.path.abspath(
+            os.path.join(__file__, "../messages/contains_invalid_messages.json")
+        )
+    )
+    with open(path) as fd:
```

Comment:
Mixing Path and os.path is weird. Also, pretty sure the join would not work because you never specify to take the parent directory of `__file__`?

```suggestion
    path = Path(__file__).parent.parent / "messages" / "contains_invalid_messages.json"
    with path.open()  as fd:
```